test/es-module/test-esm-import-meta.mjs

```diff
@@ -18,3 +18,13 @@ for (const descriptor of Object.values(descriptors)) {
 
 const urlReg = /^file:\/\/\/.*\/test\/es-module\/test-esm-import-meta\.mjs$/;
 assert(import.meta.url.match(urlReg));
+
+const dirReg = /^\/.*\/test\/es-module$/;
+assert(import.meta.__dirname.match(dirReg));
```

Comment:
```suggestion
assert.match(import.meta.__dirname, dirReg);
```

Reply:
I followed the existing assertion at https://github.com/nodejs/node/blob/b76862df0ac124f7f683dd51efd56215336d1e31/test/es-module/test-esm-import-meta.mjs#L19-L20

Do you still want my changes altered?

Reply:
yes, `assert.match` produces a better output in case of error.